addons/website_sale_delivery/static/src/js/website_sale_delivery.js

```diff
@@ -19,20 +19,41 @@ publicWidget.registry.websiteSaleDelivery = publicWidget.Widget.extend({
      * @override
      */
     start: function () {
+        var self = this;
         var $carriers = $('#delivery_carrier input[name="delivery_type"]');
         // Workaround to:
         // - update the amount/error on the label at first rendering
         // - prevent clicking on 'Pay Now' if the shipper rating fails
         if ($carriers.length > 0) {
             $carriers.filter(':checked').click();
         }
+
+        // Asynchronously retrieve every carrier price
+        _.each($carriers, function (carrierInput, k) {
+            self._showLoading($(carrierInput));
+            self._rpc({
+                route: '/shop/carrier_rate_shipment',
+                params: {
+                    carrier_id: carrierInput.value,
```

Comment:
Put `carrier_id` into quotes as this is a python-like variable :)

Reply:
done